docs/dev/contributing.rst

```diff
@@ -104,22 +93,14 @@ The following sub-sections go into more detail on some of the points above.
 
 .. _development philosophy: https://pipenv.pypa.io/en/latest/dev/philosophy/
 
-
 .. _dev-setup:
 
 Development Setup
 ~~~~~~~~~~~~~~~~~
 
-To get your development environment setup, run:
-
-.. code-block:: sh
-
-  pip install -e .
-  pipenv install --dev
+See `CONTRIBUTING.md`_ in the code repository.
```

Comment:
It is circular referencing

Reply:
I thought in the beginning only the `md` version is worth maintaining. Eventually I decided to keep the rst, and only maintain it.